agent_based: fix bounds variable update

PhilippKilian opened the pull request:
closes #19 

Update the bounds variable with "fixed" tuple.

freddy36 commented:
Seems like this was already fixed by #18.
Can you check the latest master version?

PhilippKilian committed:
agent_based: fix bounds variable update

PhilippKilian commented:
> Seems like this was already fixed by #18. Can you check the latest master version?

It's not fixed on the master. I rebased and updated my branch.
